src/resolvers/userResolver.ts

```diff
@@ -62,6 +64,25 @@ const resolvers: any = {
                 throw new ApolloError('Invalid credential', 'UserInputError')
             }
         },
+
+        async loginOrg(_: any, { orgInput: { orgName } }: any) {
+            const organisation: any = await Organisation.findOne({ orgName })
+            if (organisation) {
+                const token = jwt.sign(
+                    { orgName: organisation.orgName, },
+                    SECRET,
+                    { expiresIn: '2h' }
+                )
+                const data = {
+                    token: token,
+                    organisation: organisation.toJSON(),
```

Comment:
This can be linked to a user and be retrieved using the graphql schema. Ensuring that we just add a resolver that handles that case